Mathlib/Analysis/Normed/Lp/WithLp.lean

```diff
@@ -50,8 +51,19 @@ variable (p : ℝ≥0∞) (K : Type uK) (K' : Type uK') (V : Type uV)
 
 namespace WithLp
 
+variable {p V} in
```

Comment:
Should `p` be explicit to one or both of these?

Reply:
`p` could be explicit in `toLp`, yes

Reply:
That sounds good to me, it's quite common to want to write `‖toLp v‖`, and this won't type-check unless the `p` is specified somehow. The same problem doesn't exist for `ofLp`.